packages/ketcher-react/src/script/ui/component/form/input.tsx

```diff
@@ -314,6 +314,11 @@ function ctrlMap(component, props: Props) {
 
 function componentMap(props: Props) {
   const { schema, type, multiple } = props
+
+  if (schema?.type === 'boolean' && schema?.description === 'slider') {
```

Comment:
Could you please add tests for this function to make sure that this problem will not return later?